src/lib/navbar/navbar_display.svelte

```diff
@@ -0,0 +1,75 @@
+<!--
+  navbar_display.svelte: 
+    Manages the display/UI of the navbar, decoupled from the scrolling logic in the NavbarManager
+-->
+
+<script lang="ts">
+  /**
+   * Current step (which cat face to display, based on how far scrolled the user is)
+   */
+  export let step: number;
+
+  /**
+   * The user's current percentage (0 to 100) scrolled through the website
+   */
+  export let scrollPercentage: number;
+</script>
+
+<div class="container">
+  <div class="filled" style="width: {scrollPercentage}%" />
+  <img
+    id="cat-face-image"
+    class="cat-face"
+    src="/navbar_icons/navbar{step}.svg"
+    alt="A cat face"
+    style="left: {scrollPercentage}%"
+  />
+</div>
+
+<style>
+  .container {
+    position: fixed;
+    top: 0;
+    left: 0;
+    z-index: 999;
```

Comment:
I am not sure how to fix this because it looks like the z-index is already so large but the cat icon falls behind the cat-superhero header. 
![Screenshot 2023-06-09 124610](https://github.com/TritonSE/FIX-Landing-Page/assets/74436201/1de58698-bff4-4fdf-944c-cf92dd42ab87)

Reply:
There's another spot too where the cat paw and name go on top of the nav bar

Reply:
Is your branch up-to-date?  I set the z-index to 999 to fix the issue on my end, so I can't replicate it now.

Reply:
looks good now